web/src/people.ts

```diff
@@ -31,25 +31,25 @@ export type User = {
     // used for caching result of remove_diacritics.
     name_with_diacritics_removed?: string | undefined;
     date_joined: string;
-    is_active: boolean;
+    is_active?: boolean;
     is_owner: boolean;
     is_admin: boolean;
     is_guest: boolean;
-    is_moderator: boolean;
-    is_billing_admin: boolean;
+    is_moderator?: boolean;
+    is_billing_admin?: boolean;
     role: number;
-    timezone: string;
+    timezone?: string;
     avatar_url?: string | null;
     avatar_version: number;
-    profile_data: Record<number, ProfileDatum>;
+    profile_data?: Record<number, ProfileDatum>;
     // used for fake user objects.
     is_missing_server_data?: boolean;
     // used for inaccessible user objects.
     is_inaccessible_user?: boolean;
 } & (
     | {
           is_bot: false;
-          bot_type: null;
+          bot_type?: null;
```

Comment:
I think `is_bot` being true implied a `bot_type`...

Reply:
Oh, this is the `false` case, sure.

Reply:
This is the `is_bot: false` case.